stockflux-securities-master/src/components/Alert.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Alert.css";
+
+const Alert = ({ messages, type, className }) => {
+  return (
+    <ul className={`securities-alert-container ${className}`}>
+      {messages.map((message, index) => {
```

Comment:
Since this closure/arrow function simply returns a single parameter, you can omit the braces `{`/`}` and the `return` so it becomes:

    {messages.map((message, index) => (
      <li className={`securities-alert ${type}`} key={index}>
        {message}
      </li>
    )}

Reply:
Yep done